xbuild/src/gradle/mod.rs

```diff
@@ -102,6 +103,8 @@ pub fn build(env: &BuildEnv, libraries: Vec<(Target, PathBuf)>, out: &Path) -> R
     let _ = std::fs::remove_dir_all(&base_assets);
 
     if !config.assets.is_empty() {
+        std::fs::OpenOptions::new().write(true).append(true).open(gradle.join("settings.gradle"))?.write(r#"include ':baseAssets'"#.as_bytes())?;
```

Comment:
@MarijnS95 I have to delete the baseAssets include from settings.grade and adding it if assets are present, otherwise I get a gradle error